### PR TITLE
Turkish translations done

### PR DESCRIPTION
**Pull request by Zaryob**
Hi there.

I started localizing Turkish for image writer. I started translating en.ts file with reference. I'll let you know when all translations are finished

### CODE REVIEW

**Comment by Zaryob** [comment]
It needs to control rigth now.

File: i18n/rpi-imager_tr.ts [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+        <source>Storage capacity is not large enough.
+Needs to be at least %1 GB</source>
+        <translation>Depolama kapasitesi yeterince büyük değil. 
+En az %1 GB olması gerekiyor</translation>
+    </message>
+    <message>
+        <location filename="../imagewriter.cpp" line="176"/>
+        <source>Input file is not a valid disk image.
+File size %1 bytes is not a multiple of 512 bytes.</source>
+        <translation type="unfinished">Giriş dosyası geçerli bir disk görüntüsü değil.
```
Comment (lurch) [inline i18n/rpi-imager_tr.ts:167]: This one still says "unfinished" ?

Comment (Zaryob) [reply i18n/rpi-imager_tr.ts:167]: I put it for control. Translation is really importand. I checked Turkish Dictionary to compare my translations whether it is true

**Comment by Zaryob** [comment]
I tried up in code. Actually no problems seem

**Comment by Zaryob** [comment]
@lurch could you check it again.

**Comment by lurch** [comment]
I'm afraid I can't read Turkish, and @maxnet is in charge of this repo. But I can confirm that there aren't any "unfinished" translations in this PR now :slightly_smiling_face:

**Comment by Zaryob** [comment]
All of them complated.

**Pull request merged**